src/app/core/signing.service.ts

```diff
@@ -78,7 +78,7 @@ export class SigningService {
     const start = textbox.selectionStart;
     const end = textbox.selectionEnd;
     const text = textbox.value;
-    const scrollPos: number = textbox.scrollTop;
+    const scroll_pos: number = textbox.scrollTop;
```

Comment:
Improper variable naming
```suggestion
    const scrollPos: number = textbox.scrollTop;
```